frontends/web/src/routes/account/utils.ts

```diff
@@ -64,6 +64,9 @@ export const getCoinCode = (coinCode: CoinCode): CoinCode | undefined => {
   case 'sepeth':
     return 'eth';
   }
+  if (coinCode.startsWith('eth-erc20-')) {
+    return 'eth';
+  }
 };
```

Comment:
Looks like getCoinCode function is only used in feetargets component.

Reply:
Yep I checked as I didn't want to break other stuff :) Happy to move it somewhere else if needed. I'll merge this PR now but I can open a new one if needed!